catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/query/monitor/impl/WorkspaceServiceImplTest.java

```diff
@@ -40,48 +39,60 @@
 import org.codice.ddf.catalog.ui.metacard.workspace.WorkspaceConstants;
 import org.codice.ddf.catalog.ui.metacard.workspace.WorkspaceMetacardImpl;
 import org.codice.ddf.catalog.ui.metacard.workspace.transformer.impl.WorkspaceTransformerImpl;
-import org.codice.ddf.catalog.ui.query.monitor.api.FilterService;
 import org.codice.ddf.catalog.ui.query.monitor.api.SecurityService;
+import org.codice.ddf.catalog.ui.query.monitor.api.SubscriptionsPersistentStore;
 import org.codice.ddf.catalog.ui.query.monitor.api.WorkspaceMetacardFilter;
+import org.codice.ddf.persistence.PersistenceException;
+import org.codice.ddf.persistence.PersistentStore;
 import org.junit.Before;
 import org.junit.Test;
 import org.mockito.ArgumentCaptor;
-import org.opengis.filter.And;
 import org.opengis.filter.Filter;
-import org.opengis.filter.Or;
 
 public class WorkspaceServiceImplTest {
 
-  private CatalogFramework catalogFramework;
+  private static final String TEST_ID = "123";
 
-  private FilterBuilder filterBuilder;
+  private static final String TEST_SUBJECT = "subject";
 
-  private WorkspaceTransformerImpl workspaceTransformer;
+  private static final int MAX_SUBSCRIPTIONS = 100;
 
-  private FilterService filterService;
+  private CatalogFramework catalogFramework;
+
+  private WorkspaceTransformerImpl workspaceTransformer;
 
   private SecurityService securityService;
 
   private WorkspaceServiceImpl workspaceService;
 
+  private PersistentStore persistentStore;
+
+  private WorkspaceQueryBuilder workspaceQueryBuilder;
+
   @Before
   public void setup() {
     catalogFramework = mock(CatalogFramework.class);
-    filterBuilder = mock(FilterBuilder.class);
     workspaceTransformer = mock(WorkspaceTransformerImpl.class);
-    filterService = mock(FilterService.class);
     securityService = mock(SecurityService.class);
+    persistentStore = mock(PersistentStore.class);
```

Comment:
There are enough mock objects in here that it's worth using `@Mock` annotations instead.

Reply:
Fixed.